results.tex

```diff
@@ -0,0 +1,143 @@
+\section{Results}
+
+\subsection{Past-Predictable Future Scenario}
+
+\ref{tab:sim_result} lists the important metrics
+obtained from the first simulation. The following
+values are the EU inventory and history at year 2050.
+
+\begin{table}[h]
+	\centering
+	\label{tab:sim_result}
+	\caption {Simulation Results}
+	\scalebox{0.86}{
+		\begin{tabular}{|c|c|c|}
+			\hline
+			Category [unit] & Value & Specifics \\ \hline
+			Total UOX Usage [t] & 188,196  &  \\ \hline
+			Total MOX Usage [t] & 118 & \\ \hline
+			Total Spent UOX [t] & 183,807 & \\ \hline
+			Total Spent MOX [t] & 0 & All is Reprocessed. \\ \hline
+			Total Tailings [t] & 1,125,798 & \\ \hline
+		\end{tabular}}
+\end {table}
+
+\begin{table}[h]
+	\centering
+	\label{tab:pu}
+	\caption{Plutonium From Spent Fuel}
+	\begin{tabular}{|c|c|c|}
+		\hline
+		Isotope & Mass Fraction in Spent Fuel [\%] & Quantity [t] \\ \hline
+		Total & .9358 & 1,761.13 \\ \hline
+		Pu238 & .0111 & 20.88 \\ \
+		Pu239 & .518 & 974.85 \\ \hline
+		Pu240 & .232 & 436.61 \\ \hline
+		Pu241 & .126 & 237.12 \\ \hline
+		Pu242 & .0487 & 91.65 \\ \hline
+	\end{tabular}
+\end{table}
+
+
+To create MOX, 10\% Pu and 90\% depleted uranium is used.
+Thus $1,761.13$ tons of plutonium yields $17,611.3$ tons of
+\gls{MOX}. \ref{tab:pu} lists the isotope, mass fraction,
+and quantity that can be obtained from the 2050 \gls{SNF} inventory.
+
+
+\subsection{French \gls{SFR} Transition Scenario}
+
+From Varaine et al \cite{varaine_pre-conceptual_2012}, a French
+ASTRID-type \gls{SFR} of capacity $600 MWe$ needs $1.225$ tons of
+plutonium a year, with an initial plutonium loading of $4.9$ tons.
+
+Thus, it can be assumed that the ASTRID reactor takes in $49$ tons of 
+\gls{MOX}, and switches a quarter of its fuel every year. Simply,
+an ASTRID reactor needs $12.25$ tons of \gls{MOX} per year. 
+
+Thus the number of SFR-years worth of fuel that can be created with
+the \gls{SNF} from 2050 is $\frac{17,611.3 t}{12.25} = 1,437.65 $.
+Assuming infinite reprocessing and \gls{MOX} fabrication capacity,
+
+However, assuming that \gls{MOX} can be recycled indefinitely,
+spent \gls{MOX} contains enough plutonium for a \gls{MOX} fuel with
+the same mass, if mixed with depleted uranium.
```

Comment:
I tried to explain below, though 38% plutonium does seem a bit too much, but it does have a high burnup:
```
However, assuming that \gls{MOX} can be recycled indefinitely,
spent \gls{MOX} from an ASTRID reactor
contains enough plutonium to produce a \gls{MOX} fuel with
the same mass, if mixed with depleted uranium. For example,
spent \gls{MOX} from an ASTRID reactor is 38\% plutonium,
whereas a fresh \gls{MOX} is 10\% plutonium. Separating plutonium
from spent \gls{MOX} from an ASTRID reactor can create \gls{MOX} more than
three times the mass of spent \gls{MOX}.
```